frontend/src/components/Inputs/Job.NEW

```diff
@@ -0,0 +1,40 @@
+<template>
```

Comment:
Interesting file!?
Why is it in the PR, we don't use it yet... Not sure how to handle this!